<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="7" ignored="28" total="58" passed="19">
  <reporter-output>
  </reporter-output>
  <suite name="TestAll" duration-ms="412409" started-at="2017-03-28T15:00:33Z" finished-at="2017-03-28T15:07:25Z">
    <groups>
      <group name="negative">
        <method signature="LoginTest.testSignin_IncorrectCredentials()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0]" name="testSignin_IncorrectCredentials" class="com.lohika.gap.tests.LoginTest"/>
        <method signature="LoginTest.testSignin_IncorrectCredentials()[pri:0, instance:com.lohika.gap.tests.LoginTest@13a57a3b]" name="testSignin_IncorrectCredentials" class="com.lohika.gap.tests.LoginTest"/>
        <method signature="ChangeSettingsTest.testAddNewEmail_AlreadyExist()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@69d9c55]" name="testAddNewEmail_AlreadyExist" class="com.lohika.gap.tests.ChangeSettingsTest"/>
        <method signature="ChangeSettingsTest.testChangePassword_WrongNewPassword()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@69d9c55]" name="testChangePassword_WrongNewPassword" class="com.lohika.gap.tests.ChangeSettingsTest"/>
        <method signature="ChangeSettingsTest.testChangePassword_WrongOldPassword()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@69d9c55]" name="testChangePassword_WrongOldPassword" class="com.lohika.gap.tests.ChangeSettingsTest"/>
        <method signature="ManageRepositoriesTest.testCreateNewRepository_AlreadyExist()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@26f67b76]" name="testCreateNewRepository_AlreadyExist" class="com.lohika.gap.tests.ManageRepositoriesTest"/>
        <method signature="ManageRepositoriesTest.testCreateNewRepository_NoName()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@26f67b76]" name="testCreateNewRepository_NoName" class="com.lohika.gap.tests.ManageRepositoriesTest"/>
        <method signature="ManageRepositoriesTest.testRenameRepository_NameAlreadyExist()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@26f67b76]" name="testRenameRepository_NameAlreadyExist" class="com.lohika.gap.tests.ManageRepositoriesTest"/>
        <method signature="RepositoryDescriptionTest.testSetNewDescription_IncorectWebpage()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@153f5a29]" name="testSetNewDescription_IncorectWebpage" class="com.lohika.gap.tests.RepositoryDescriptionTest"/>
        <method signature="RepositoryFilesTest.testAddNewFile_AlreadyExists()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@7f560810]" name="testAddNewFile_AlreadyExists" class="com.lohika.gap.tests.RepositoryFilesTest"/>
        <method signature="RepositoryFilesTest.testAddNewFile_Canceled()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@7f560810]" name="testAddNewFile_Canceled" class="com.lohika.gap.tests.RepositoryFilesTest"/>
        <method signature="RepositoryFilesTest.testUploadFile_Failed()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@7f560810]" name="testUploadFile_Failed" class="com.lohika.gap.tests.RepositoryFilesTest"/>
      </group> <!-- negative -->
      <group name="positive">
        <method signature="ChangeSettingsTest.testAddNewEmail()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="testAddNewEmail" class="com.lohika.gap.tests.ChangeSettingsTest"/>
        <method signature="ChangeSettingsTest.testChangeEmailPreferences()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="testChangeEmailPreferences" class="com.lohika.gap.tests.ChangeSettingsTest"/>
        <method signature="ChangeSettingsTest.testChangeUserProfile()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="testChangeUserProfile" class="com.lohika.gap.tests.ChangeSettingsTest"/>
        <method signature="ChangeSettingsTest.testRemoveEmail()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="testRemoveEmail" class="com.lohika.gap.tests.ChangeSettingsTest"/>
        <method signature="LoginTest.testSignout()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0]" name="testSignout" class="com.lohika.gap.tests.LoginTest"/>
        <method signature="LoginTest.testSignin()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0]" name="testSignin" class="com.lohika.gap.tests.LoginTest"/>
        <method signature="ManageRepositoriesTest.testCreateNewRepository()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464]" name="testCreateNewRepository" class="com.lohika.gap.tests.ManageRepositoriesTest"/>
        <method signature="ManageRepositoriesTest.testRemoveRepository()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464]" name="testRemoveRepository" class="com.lohika.gap.tests.ManageRepositoriesTest"/>
        <method signature="RepositoryDescriptionTest.testRemoveCurrentDescription()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@617c74e5]" name="testRemoveCurrentDescription" class="com.lohika.gap.tests.RepositoryDescriptionTest"/>
        <method signature="RepositoryDescriptionTest.testSetNewDescription()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@617c74e5]" name="testSetNewDescription" class="com.lohika.gap.tests.RepositoryDescriptionTest"/>
        <method signature="RepositoryDescriptionTest.testSetNewDescription_Canceled()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@617c74e5]" name="testSetNewDescription_Canceled" class="com.lohika.gap.tests.RepositoryDescriptionTest"/>
        <method signature="RepositoryFilesTest.testAddNewFile()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testAddNewFile" class="com.lohika.gap.tests.RepositoryFilesTest"/>
        <method signature="RepositoryFilesTest.testEditFile()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testEditFile" class="com.lohika.gap.tests.RepositoryFilesTest"/>
        <method signature="RepositoryFilesTest.testRemoveFile()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testRemoveFile" class="com.lohika.gap.tests.RepositoryFilesTest"/>
        <method signature="RepositoryFilesTest.testUploadFile()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testUploadFile" class="com.lohika.gap.tests.RepositoryFilesTest"/>
        <method signature="RepositoryFilesTest.testUploadFileViaAUTOIT()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testUploadFileViaAUTOIT" class="com.lohika.gap.tests.RepositoryFilesTest"/>
        <method signature="ManageRepositoriesTest.testRenameRepository()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464]" name="testRenameRepository" class="com.lohika.gap.tests.ManageRepositoriesTest"/>
        <method signature="LoginTest.testSignout()[pri:0, instance:com.lohika.gap.tests.LoginTest@13a57a3b]" name="testSignout" class="com.lohika.gap.tests.LoginTest"/>
      </group> <!-- positive -->
    </groups>
    <test name="All_positive" duration-ms="202244" started-at="2017-03-28T15:00:33Z" finished-at="2017-03-28T15:03:55Z">
      <class name="com.lohika.gap.tests.RepositoryFilesTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="SetupMethod" is-config="true" duration-ms="8545" started-at="2017-03-28T18:02:10Z" finished-at="2017-03-28T18:02:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="PASS" signature="testAddNewFile()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testAddNewFile" duration-ms="8694" started-at="2017-03-28T18:02:19Z" finished-at="2017-03-28T18:02:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddNewFile -->
        <test-method status="PASS" signature="testEditFile()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testEditFile" duration-ms="9455" started-at="2017-03-28T18:02:28Z" finished-at="2017-03-28T18:02:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEditFile -->
        <test-method status="PASS" signature="testUploadFile()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testUploadFile" duration-ms="13431" started-at="2017-03-28T18:02:37Z" finished-at="2017-03-28T18:02:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUploadFile -->
        <test-method status="PASS" signature="testUploadFileViaAUTOIT()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testUploadFileViaAUTOIT" duration-ms="13932" started-at="2017-03-28T18:02:51Z" finished-at="2017-03-28T18:03:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUploadFileViaAUTOIT -->
        <test-method status="PASS" signature="testRemoveFile()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="testRemoveFile" duration-ms="6324" started-at="2017-03-28T18:03:05Z" depends-on-methods="com.lohika.gap.tests.RepositoryFilesTest.testAddNewFile" finished-at="2017-03-28T18:03:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRemoveFile -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@6537cf78]" name="cleanMethod" is-config="true" duration-ms="823" started-at="2017-03-28T18:03:11Z" finished-at="2017-03-28T18:03:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.RepositoryFilesTest -->
      <class name="com.lohika.gap.tests.LoginTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0]" name="SetupMethod" is-config="true" duration-ms="8644" started-at="2017-03-28T18:03:43Z" finished-at="2017-03-28T18:03:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="FAIL" signature="testSignout()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0]" name="testSignout" duration-ms="3034" started-at="2017-03-28T18:03:51Z" finished-at="2017-03-28T18:03:54Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[User is not signed out expected [The world's leading software development platform В· GitHub] but found [The world's leading software development platform · GitHub]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: User is not signed out expected [The world's leading software development platform В· GitHub] but found [The world's leading software development platform · GitHub]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.lohika.gap.tests.LoginTest.testSignout(LoginTest.java:13)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSignout -->
        <test-method status="SKIP" signature="testSignin()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0]" name="testSignin" duration-ms="0" started-at="2017-03-28T18:03:54Z" depends-on-methods="com.lohika.gap.tests.LoginTest.testSignin_IncorrectCredentials" finished-at="2017-03-28T18:03:54Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginTest.testSignin()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginTest.testSignin()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSignin -->
        <test-method status="SKIP" signature="testSignin_IncorrectCredentials()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0]" name="testSignin_IncorrectCredentials" duration-ms="0" started-at="2017-03-28T18:03:54Z" depends-on-methods="com.lohika.gap.tests.LoginTest.testSignout" finished-at="2017-03-28T18:03:54Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginTest.testSignin_IncorrectCredentials()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginTest.testSignin_IncorrectCredentials()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSignin_IncorrectCredentials -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.LoginTest@34b7bfc0]" name="cleanMethod" is-config="true" duration-ms="704" started-at="2017-03-28T18:03:54Z" finished-at="2017-03-28T18:03:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.LoginTest -->
      <class name="com.lohika.gap.tests.ManageRepositoriesTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464]" name="SetupMethod" is-config="true" duration-ms="9482" started-at="2017-03-28T18:00:33Z" finished-at="2017-03-28T18:00:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="FAIL" signature="testCreateNewRepository()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464]" name="testCreateNewRepository" duration-ms="63117" started-at="2017-03-28T18:00:42Z" finished-at="2017-03-28T18:01:45Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[chrome not reachable
  (Session info: chrome=56.0.2924.87)
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4.02 seconds
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'vova-pc', ip: '172.22.90.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_73'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9), userDataDir=C:\Users\vova\AppData\Local\Temp\scoped_dir5836_1275}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=56.0.2924.87, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 67a7615041e6dc6bb277cb4e66c3b8fd
*** Element info: {Using=xpath, value=//button[@data-disable-with='Creating repositoryвЂ¦']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: chrome not reachable
  (Session info: chrome=56.0.2924.87)
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4.02 seconds
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'vova-pc', ip: '172.22.90.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_73'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9), userDataDir=C:\Users\vova\AppData\Local\Temp\scoped_dir5836_1275}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=56.0.2924.87, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 67a7615041e6dc6bb277cb4e66c3b8fd
*** Element info: {Using=xpath, value=//button[@data-disable-with='Creating repositoryвЂ¦']}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:216)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:168)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:635)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:368)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:360)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy5.toString(Unknown Source)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at com.lohika.gap.core.Wait.forElement(Wait.java:40)
	at com.lohika.gap.core.BasePage.clickOnElement(BasePage.java:27)
	at com.lohika.gap.pages.NewRepositoryPage.createNewRepository(NewRepositoryPage.java:23)
	at com.lohika.gap.tests.ManageRepositoriesTest.testCreateNewRepository(ManageRepositoriesTest.java:11)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateNewRepository -->
        <test-method status="FAIL" signature="testRenameRepository()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464]" name="testRenameRepository" duration-ms="4037" started-at="2017-03-28T18:01:45Z" finished-at="2017-03-28T18:01:50Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[chrome not reachable
  (Session info: chrome=56.0.2924.87)
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4.04 seconds
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'vova-pc', ip: '172.22.90.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_73'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9), userDataDir=C:\Users\vova\AppData\Local\Temp\scoped_dir5836_1275}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=56.0.2924.87, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 67a7615041e6dc6bb277cb4e66c3b8fd]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: chrome not reachable
  (Session info: chrome=56.0.2924.87)
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 4.04 seconds
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'vova-pc', ip: '172.22.90.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_73'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9), userDataDir=C:\Users\vova\AppData\Local\Temp\scoped_dir5836_1275}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=56.0.2924.87, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: 67a7615041e6dc6bb277cb4e66c3b8fd
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:216)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:168)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:635)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:658)
	at org.openqa.selenium.remote.RemoteWebDriver.getCurrentUrl(RemoteWebDriver.java:332)
	at com.lohika.gap.pages.HomePage.selectRepository(HomePage.java:41)
	at com.lohika.gap.tests.ManageRepositoriesTest.testRenameRepository(ManageRepositoriesTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRenameRepository -->
        <test-method status="SKIP" signature="testRemoveRepository()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464]" name="testRemoveRepository" duration-ms="0" started-at="2017-03-28T18:01:50Z" depends-on-methods="com.lohika.gap.tests.ManageRepositoriesTest.testCreateNewRepository" finished-at="2017-03-28T18:01:50Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method ManageRepositoriesTest.testRemoveRepository()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method ManageRepositoriesTest.testRemoveRepository()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRemoveRepository -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@3a03464]" name="cleanMethod" is-config="true" duration-ms="811" started-at="2017-03-28T18:01:50Z" finished-at="2017-03-28T18:01:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.ManageRepositoriesTest -->
      <class name="com.lohika.gap.tests.RepositoryDescriptionTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@617c74e5]" name="SetupMethod" is-config="true" duration-ms="8107" started-at="2017-03-28T18:01:50Z" finished-at="2017-03-28T18:01:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="PASS" signature="testRemoveCurrentDescription()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@617c74e5]" name="testRemoveCurrentDescription" duration-ms="5182" started-at="2017-03-28T18:01:58Z" finished-at="2017-03-28T18:02:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRemoveCurrentDescription -->
        <test-method status="PASS" signature="testSetNewDescription()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@617c74e5]" name="testSetNewDescription" duration-ms="3895" started-at="2017-03-28T18:02:04Z" finished-at="2017-03-28T18:02:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetNewDescription -->
        <test-method status="PASS" signature="testSetNewDescription_Canceled()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@617c74e5]" name="testSetNewDescription_Canceled" duration-ms="2275" started-at="2017-03-28T18:02:08Z" finished-at="2017-03-28T18:02:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetNewDescription_Canceled -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@617c74e5]" name="cleanMethod" is-config="true" duration-ms="661" started-at="2017-03-28T18:02:10Z" finished-at="2017-03-28T18:02:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.RepositoryDescriptionTest -->
      <class name="com.lohika.gap.tests.ChangeSettingsTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="SetupMethod" is-config="true" duration-ms="8349" started-at="2017-03-28T18:03:12Z" finished-at="2017-03-28T18:03:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="PASS" signature="testAddNewEmail()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="testAddNewEmail" duration-ms="4334" started-at="2017-03-28T18:03:20Z" finished-at="2017-03-28T18:03:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddNewEmail -->
        <test-method status="PASS" signature="testChangeEmailPreferences()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="testChangeEmailPreferences" duration-ms="6556" started-at="2017-03-28T18:03:24Z" finished-at="2017-03-28T18:03:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testChangeEmailPreferences -->
        <test-method status="FAIL" signature="testChangeUserProfile()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="testChangeUserProfile" duration-ms="5996" started-at="2017-03-28T18:03:31Z" finished-at="2017-03-28T18:03:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Profile wasn't updated expected [Profile updated successfully вЂ” view your profile.] but found [Profile updated successfully — view your profile.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Profile wasn't updated expected [Profile updated successfully вЂ” view your profile.] but found [Profile updated successfully — view your profile.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.lohika.gap.tests.ChangeSettingsTest.testChangeUserProfile(ChangeSettingsTest.java:12)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testChangeUserProfile -->
        <test-method status="PASS" signature="testRemoveEmail()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="testRemoveEmail" duration-ms="5082" started-at="2017-03-28T18:03:37Z" depends-on-methods="com.lohika.gap.tests.ChangeSettingsTest.testAddNewEmail" finished-at="2017-03-28T18:03:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRemoveEmail -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@67b6d4ae]" name="cleanMethod" is-config="true" duration-ms="693" started-at="2017-03-28T18:03:42Z" finished-at="2017-03-28T18:03:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.ChangeSettingsTest -->
    </test> <!-- All_positive -->
    <test name="All_negative" duration-ms="210088" started-at="2017-03-28T15:03:55Z" finished-at="2017-03-28T15:07:25Z">
      <class name="com.lohika.gap.tests.RepositoryFilesTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@7f560810]" name="SetupMethod" is-config="true" duration-ms="8227" started-at="2017-03-28T18:06:21Z" finished-at="2017-03-28T18:06:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="PASS" signature="testAddNewFile_AlreadyExists()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@7f560810]" name="testAddNewFile_AlreadyExists" duration-ms="6379" started-at="2017-03-28T18:06:29Z" finished-at="2017-03-28T18:06:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddNewFile_AlreadyExists -->
        <test-method status="PASS" signature="testAddNewFile_Canceled()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@7f560810]" name="testAddNewFile_Canceled" duration-ms="10520" started-at="2017-03-28T18:06:36Z" finished-at="2017-03-28T18:06:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddNewFile_Canceled -->
        <test-method status="PASS" signature="testUploadFile_Failed()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@7f560810]" name="testUploadFile_Failed" duration-ms="5161" started-at="2017-03-28T18:06:46Z" finished-at="2017-03-28T18:06:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUploadFile_Failed -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.RepositoryFilesTest@7f560810]" name="cleanMethod" is-config="true" duration-ms="654" started-at="2017-03-28T18:06:51Z" finished-at="2017-03-28T18:06:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.RepositoryFilesTest -->
      <class name="com.lohika.gap.tests.LoginTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.LoginTest@13a57a3b]" name="SetupMethod" is-config="true" duration-ms="7916" started-at="2017-03-28T18:07:14Z" finished-at="2017-03-28T18:07:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="FAIL" signature="testSignout()[pri:0, instance:com.lohika.gap.tests.LoginTest@13a57a3b]" name="testSignout" duration-ms="2682" started-at="2017-03-28T18:07:22Z" finished-at="2017-03-28T18:07:25Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[User is not signed out expected [The world's leading software development platform В· GitHub] but found [The world's leading software development platform · GitHub]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: User is not signed out expected [The world's leading software development platform В· GitHub] but found [The world's leading software development platform · GitHub]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.lohika.gap.tests.LoginTest.testSignout(LoginTest.java:13)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSignout -->
        <test-method status="SKIP" signature="testSignin_IncorrectCredentials()[pri:0, instance:com.lohika.gap.tests.LoginTest@13a57a3b]" name="testSignin_IncorrectCredentials" duration-ms="0" started-at="2017-03-28T18:07:25Z" depends-on-methods="com.lohika.gap.tests.LoginTest.testSignout" finished-at="2017-03-28T18:07:25Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginTest.testSignin_IncorrectCredentials()[pri:0, instance:com.lohika.gap.tests.LoginTest@13a57a3b] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginTest.testSignin_IncorrectCredentials()[pri:0, instance:com.lohika.gap.tests.LoginTest@13a57a3b] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSignin_IncorrectCredentials -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.LoginTest@13a57a3b]" name="cleanMethod" is-config="true" duration-ms="682" started-at="2017-03-28T18:07:25Z" finished-at="2017-03-28T18:07:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.LoginTest -->
      <class name="com.lohika.gap.tests.ManageRepositoriesTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@26f67b76]" name="SetupMethod" is-config="true" duration-ms="8155" started-at="2017-03-28T18:03:55Z" finished-at="2017-03-28T18:04:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="FAIL" signature="testCreateNewRepository_AlreadyExist()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@26f67b76]" name="testCreateNewRepository_AlreadyExist" duration-ms="59337" started-at="2017-03-28T18:04:03Z" finished-at="2017-03-28T18:05:03Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 10s. waiting for element: Proxy element for: DefaultElementLocator 'By.xpath: //button[@data-disable-with='Creating repositoryвЂ¦']'
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'vova-pc', ip: '172.22.90.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_73'
Driver info: driver.version: unknown]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 10s. waiting for element: Proxy element for: DefaultElementLocator 'By.xpath: //button[@data-disable-with='Creating repositoryвЂ¦']'
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'vova-pc', ip: '172.22.90.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_73'
Driver info: driver.version: unknown
	at com.lohika.gap.core.Wait.forElement(Wait.java:43)
	at com.lohika.gap.core.BasePage.clickOnElement(BasePage.java:27)
	at com.lohika.gap.pages.NewRepositoryPage.createNewRepository(NewRepositoryPage.java:23)
	at com.lohika.gap.tests.ManageRepositoriesTest.testCreateNewRepository_AlreadyExist(ManageRepositoriesTest.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateNewRepository_AlreadyExist -->
        <test-method status="FAIL" signature="testCreateNewRepository_NoName()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@26f67b76]" name="testCreateNewRepository_NoName" duration-ms="59727" started-at="2017-03-28T18:05:03Z" finished-at="2017-03-28T18:06:02Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 10s. waiting for element: Proxy element for: DefaultElementLocator 'By.xpath: //button[@data-disable-with='Creating repositoryвЂ¦']'
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'vova-pc', ip: '172.22.90.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_73'
Driver info: driver.version: unknown]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 10s. waiting for element: Proxy element for: DefaultElementLocator 'By.xpath: //button[@data-disable-with='Creating repositoryвЂ¦']'
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'vova-pc', ip: '172.22.90.81', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_73'
Driver info: driver.version: unknown
	at com.lohika.gap.core.Wait.forElement(Wait.java:43)
	at com.lohika.gap.core.BasePage.clickOnElement(BasePage.java:27)
	at com.lohika.gap.pages.NewRepositoryPage.createNewRepository(NewRepositoryPage.java:23)
	at com.lohika.gap.tests.ManageRepositoriesTest.testCreateNewRepository_NoName(ManageRepositoriesTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:281)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:121)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:290)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:242)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreateNewRepository_NoName -->
        <test-method status="PASS" signature="testRenameRepository_NameAlreadyExist()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@26f67b76]" name="testRenameRepository_NameAlreadyExist" duration-ms="5915" started-at="2017-03-28T18:06:02Z" finished-at="2017-03-28T18:06:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRenameRepository_NameAlreadyExist -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.ManageRepositoriesTest@26f67b76]" name="cleanMethod" is-config="true" duration-ms="776" started-at="2017-03-28T18:06:08Z" finished-at="2017-03-28T18:06:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.ManageRepositoriesTest -->
      <class name="com.lohika.gap.tests.RepositoryDescriptionTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@153f5a29]" name="SetupMethod" is-config="true" duration-ms="8195" started-at="2017-03-28T18:06:09Z" finished-at="2017-03-28T18:06:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="PASS" signature="testSetNewDescription_IncorectWebpage()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@153f5a29]" name="testSetNewDescription_IncorectWebpage" duration-ms="3264" started-at="2017-03-28T18:06:17Z" finished-at="2017-03-28T18:06:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetNewDescription_IncorectWebpage -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.RepositoryDescriptionTest@153f5a29]" name="cleanMethod" is-config="true" duration-ms="682" started-at="2017-03-28T18:06:21Z" finished-at="2017-03-28T18:06:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.RepositoryDescriptionTest -->
      <class name="com.lohika.gap.tests.ChangeSettingsTest">
        <test-method status="PASS" signature="SetupMethod()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@69d9c55]" name="SetupMethod" is-config="true" duration-ms="8033" started-at="2017-03-28T18:06:52Z" finished-at="2017-03-28T18:07:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetupMethod -->
        <test-method status="PASS" signature="testAddNewEmail_AlreadyExist()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@69d9c55]" name="testAddNewEmail_AlreadyExist" duration-ms="4480" started-at="2017-03-28T18:07:00Z" finished-at="2017-03-28T18:07:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAddNewEmail_AlreadyExist -->
        <test-method status="PASS" signature="testChangePassword_WrongNewPassword()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@69d9c55]" name="testChangePassword_WrongNewPassword" duration-ms="4536" started-at="2017-03-28T18:07:05Z" finished-at="2017-03-28T18:07:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testChangePassword_WrongNewPassword -->
        <test-method status="PASS" signature="testChangePassword_WrongOldPassword()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@69d9c55]" name="testChangePassword_WrongOldPassword" duration-ms="4103" started-at="2017-03-28T18:07:09Z" finished-at="2017-03-28T18:07:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testChangePassword_WrongOldPassword -->
        <test-method status="PASS" signature="cleanMethod()[pri:0, instance:com.lohika.gap.tests.ChangeSettingsTest@69d9c55]" name="cleanMethod" is-config="true" duration-ms="650" started-at="2017-03-28T18:07:13Z" finished-at="2017-03-28T18:07:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanMethod -->
      </class> <!-- com.lohika.gap.tests.ChangeSettingsTest -->
    </test> <!-- All_negative -->
  </suite> <!-- TestAll -->
</testng-results>
